cluster/cluster.go

```diff
@@ -17,8 +17,9 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	rpcClient "github.com/weaviate/weaviate/cluster/rpc/client"
+	rpcServer "github.com/weaviate/weaviate/cluster/rpc/server"
 	"github.com/weaviate/weaviate/cluster/store"
```

Comment:
**thought:** wdyt if we put `client` and `server` in the same package to use them as `rpc.Client` , `rpc.Server` ?

Reply:
Agree